HACKING.md

```diff
@@ -392,24 +436,34 @@ source files if `run-checks` complains about the format.
 You can run an individual test for a sub-package by changing into that 
 directory and:
 
-    go test -check.f $testname
+```
+LANG=C.UTF-8 go test -check.f $testname
```

Comment:
mmm... is `.UTF-8` needed?

Reply:
@mardy , as a person who lives in GB and has a system locale that is not US, I assure you that this one is critical. :-) When it fails for someone (especially fresh) who has a different locale, finding the reason is even far beyond from the definition of "painful", especially if you have no debugger (which I hope to resolve soon too:-)).  

By the way, I've added a comment explaining the reasoning.

Reply:
@are-scenic it would be great if you can explain which tests fail and how (not in this PR), we should probably fix those tests so that they work on all locales (i.e. the test itself sets a specific expected locale when executing) without having to specify this

Reply:
Sure I can. I will identify them and provide you with the list.